docs/milestone-2-requirements.md

```diff
@@ -0,0 +1,39 @@
+# Milestone 2
+
+## Context
+
+In the previous milestone, we have been able to provision a Storage Tenant to
+a VMware vCloud Director Tenant Organization.
+
+The goal of this milestone is to provide the basic storage operations to
+a VMware vCloud Director Tenant Organization.
+
+## Requirements
+
+### Stories
+
+Basic buckets operations:
+
+**As a** vCloud Organization Administrator
+
+**I want** to list, create, and delete buckets using vCloud Director Tenant UI
+
+**So that** I will be able to have basic buckets operations on vCloud Director UI
+
+Basic objects operations:
+
+**As a** vCloud Organization Administrator
+
+**I want** to list, upload, and download objects using vCloud Director Tenant UI
+
+**So that** I will be able to have basic objects operations on vCloud Director UI
+
+### Acceptance criteria
+
+The acceptance criteria are:
+
```

Comment:
We need documentation as well to configure S3 component as well.

Reply:
I disagree, we shouldn't need a specific configuration on Cloud Server to integrate with the Object Storage Extension.

I added an acceptance criterion related to the object storage extension configuration required to integrate with a CloudServer.

Reply:
cool :)

Reply:
we can refer to OSE documentation for how to configure S3 service.